wiki/People/Support_Team/en.md

```diff
@@ -1,20 +1,20 @@
 Support Team
-===============
+============
 
-The **Support Team** (also known as **Support Team Redux**, not to be confused with the account support team, colloquially known as the "Support Team") are members of the osu! staff whom are focused at forum moderation on several subforums: [Development](https://osu.ppy.sh/forum/2), [Gameplay & Rankings](https://osu.ppy.sh/forum/13) (excluding [Tournaments](https://osu.ppy.sh/forum/55) and [Mapping Techniques](https://osu.ppy.sh/forum/61)), [Skinning](https://osu.ppy.sh/forum/15), [Feature Requests](https://osu.ppy.sh/forum/4), and [Help](https://osu.ppy.sh/forum/5).
+The **Support Team** (also known as **Support Team Redux**, not to be confused with the account support team, also known as the "Support Team") are members of the osu! staff whom are focused at forum moderation on several subforums: [Development](https://osu.ppy.sh/forum/2), [Gameplay & Rankings](https://osu.ppy.sh/forum/13) (excluding [Tournaments](https://osu.ppy.sh/forum/55) and [Mapping Techniques](https://osu.ppy.sh/forum/61)), [Skinning](https://osu.ppy.sh/forum/15), [Feature Requests](https://osu.ppy.sh/forum/4), and [Help](https://osu.ppy.sh/forum/5).
 
 Responsibilities
--------------------
+----------------
 
 The Support Team is responsible on:
 
 1.  Testing and/or confirming bug reports to let the developers know and for them to resolve.
-2.  Organizing threads at [Feature Requests](https://osu.ppy.sh/forum/4) and [Help](https://osu.ppy.sh/forum/5) and marking them as Resolved, Confirmed, Invalid, Duplicate and Added.
+2.  Organizing threads at [Feature Requests](https://osu.ppy.sh/forum/4) and [Help](https://osu.ppy.sh/forum/5), marking them as Resolved, Confirmed, Invalid, Duplicate and Added.
```

Comment:
suggestion:
`2.  Organizing threads at [Feature Requests](https://osu.ppy.sh/forum/4) and [Help](https://osu.ppy.sh/forum/5), marking them as _Resolved_, _Confirmed_, _Invalid_, _Duplicate_, and/or _Added_.`